### ISSUE TITLE
Conda package

### ISSUE THREAD

**Issue by jthorton** [isayevlab/AIMNet2#13]
Hi, thanks for making these fantastic models open-source. Is there any interest in making AIMNet2 conda installable similar to torchani to make it easier to install and use? I am more than happy to help or do this.

**Comment by jthorton** [comment]
Some thoughts on how this could work:
- make this package conda installable and distribute all of the model weights with the package
- Move the model weights to another package or to be a release artifact and have the package download them when the model is needed similar to the torchani method.

**Comment by isayev** [comment]
Dear @jthorton thanks for the suggestions. I agree that this repo needs to be restructured. I will create a separate repo for the model zoo like in Torchani!

**Comment by jthorton** [comment]
Hi @isayev thanks for setting that up, would you like me to make a PR to move the model weights?

**Comment by isayev** [comment]
I will move model weights, thanks!

**Comment by corinwagen** [comment]
agreed here - particularly w/ the extra dependencies added w/ PBC code, would be lovely to have a conda package!
